pegasus/helpers/section_api_helpers.rb

```diff
@@ -206,13 +206,20 @@ def self.valid_courses(user_id = nil)
            all.
            map do |course|
              name = course[:name]
+             category = ScriptConstants.category(name) || 'other'
              if name == ScriptConstants::MINECRAFT_NAME
                name = ScriptConstants::MINECRAFT_TEACHER_DASHBOARD_NAME
              elsif name == ScriptConstants::HOC_NAME
                name = ScriptConstants::HOC_TEACHER_DASHBOARD_NAME
              end
```

Comment:
This logic feels like it should belong in the `ScriptConstants` module as a `teacher_dashboard_category` method or some such.